src/main/java/com/microfocus/octane/plugins/descriptors/OctaneEntityTypeDescriptor.java

```diff
@@ -104,7 +106,11 @@ public String buildEntityUrl(String baseUrl, long spaceId, long workspaceId, Str
     }
 
     public String buildTestTabEntityUrl(String baseUrl, long spaceId, long workspaceId, String entityId, String subtype) {
```

Comment:
I would just pass the spaceConfiguration from CoverageUIHelper if you need it, since it's redundant to pass spaceConfig.getBaseUrl as parameter and then get the space config again based on baseUrl